c8/client.py

```diff
@@ -26,9 +26,9 @@ class C8Client(object):
 
     def __init__(
         self,
-        protocol="http",
+        protocol="https",
```

Comment:
Thanks @the-jesus-aviles for this! We are glad you are working our python sdk! We encourage you to keep working with it and do let us know whenever you face any issue!